src/Fragments/GoogleMaps.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\ContentSecurity\Fragments;
+
+use Silverstripe\CSP\Directive;
+use Silverstripe\CSP\Fragments\Fragment;
+use Silverstripe\CSP\Policies\Policy;
+
+/*
+ * Allows execution of Google Maps API related resources
+ * In most cases Google Maps API calls are asynchronous thus can be linked and nonced accordingly.
+ * In the cases of including Google Maps API calls via typescript this fragment will be required
```

Comment:
Could you add some more details as to why this is required for typescript? Not sure I understand the distinction here

Reply:
Yeah I don't know what I was thinking there,  that doesn't make sense at all 🤣 

Its the `connect-src` that is the issue, added in the `img-src` directive in the cases where developers only need Google Maps API csp.

Removed that and updated the comment.

Reply:
Additionally I'm quite happy to leave fragments like this out of the module and move them to the project but I thought Google Maps API is fairly common enough to be included as a base.